### PR TITLE
add editsnipe command + tiny stuff

### PR DESCRIPTION
**Pull request by whirlxd**
## What is the purpose of this pr
- Added another command that is being removed ( editsnipe ) & just some tiny stuff which don't impact the bot in any significant way
-

### CODE REVIEW

**Comment by melmsie** [comment]
PM2 shouldn't be forced, so people can use whatever thing they want. You're welcome to add something to the readme suggesting PM2 however. Remove PM2 and make sure to lint and we'll accept this PR.

File: README.md [hunk 1]
```diff
@@ -2,33 +2,33 @@
...
 ```
 
 2. Create config.json:
 
-```
+```json
 {
-	"token": "<Your bot's token>",
-	"application_id": "<Your application's id>"
+	"token": "<Your bot's token>", //obtain from https://discord.com/developers/applications
```
Comment (badosz0) [inline README.md:22]: I don't think it's right place for these comments, especially with github highlighting.

File: src/index.js [hunk 2]
```diff
@@ -1,40 +1,69 @@
...
-	};
+client.on("messageDelete", async message => {
+  snipes[message.guild.id] = {
+    author: message.author,
+    content: message.content,
+    channel: message.channel.id,
+    createdAt: message.createdTimestamp,
+  };
+});
+client.on("messageUpdate", async (OldMessage, NewMessage) => {
```
Comment (badosz0) [inline src/index.js:22]: OldMessage and NewMessage should be in pascalCase.

File: src/index.js [hunk 3]
```diff
@@ -1,40 +1,69 @@
...
+      await interaction.reply(
+        snipe
+          ? {
+              embeds: [
+                new MessageEmbed()
+                  .setDescription(
+                    `Old Message:\n${snipe.oldcontent}\nNewMessage:${snipe.newcontent}`
+                  )
+                  .setAuthor(snipe.author.tag)
+                  .setFooter(`Sniped from : <#${snipe.channel}`),
```
Comment (badosz0) [inline src/index.js:60]: Missing ">".

File: src/index.js [hunk 4]
```diff
@@ -1,40 +1,69 @@
...
+      const snipe = snipes[interaction.guildId];
+
+      await interaction.reply(
+        snipe
+          ? {
+              embeds: [
+                new MessageEmbed()
+                  .setDescription(snipe.content)
+                  .setAuthor(snipe.author.tag)
+                  .setFooter(`Sniped from : <#${snipe.channel}`)
```
Comment (badosz0) [inline src/index.js:42]: Missing ">".

File: scripts/register.js [hunk 5]
```diff
@@ -2,35 +2,38 @@ const { REST } = require("@discordjs/rest");
...
-		await rest.put(
-			guild
-				? Routes.applicationGuildCommands(application_id, guild)
-				: Routes.applicationCommands(application_id),
-			{
-				body: commands,
-			}
-		);
+    await rest.put(
+      //if there is the guild id then register guild specific sommands otherwise register a global command.
```
Comment (badosz0) [inline scripts/register.js:26]: Not sure if this comment is needed here, this is pretty self explanatory.

Comment (whirlxd) [reply scripts/register.js:26]: alright it's there for when you are a newbie [ who not really familiar with the new es6 and stuff ]  and are just looking at code

**Comment by whirlxd** [comment]
i'll close this pr and just update it once again with linting aswell ( i suppose i need to use eslint )

**Pull request closed without merging**